examples/_quick-start/python/example.py

```diff
@@ -8,9 +8,10 @@
 client = hvac.Client(
     url='http://127.0.0.1:8200',
     token='dev-only-token',
+    namespace='admin',  # If namespace is configured, make sure to set here
```

Comment:
think `admin` namespace is a special child, think we can comment this out?
```suggestion
    # namespace='admin',  # If namespace is configured, make sure to set here
```

Reply:
These examples are used on our [Developer Quickstart](https://developer.hashicorp.com/vault/docs/get-started/developer-qs) page, which currently is written to test against a locally running dev Vault server. So the easiest way to not rewrite the whole quickstart doc right now would probably just be to include a comment like this: 

```
# On HCP Vault clusters, all secrets live within a top-level 'admin' namespace. This can be specified with namespace='admin' when initializing the client.
client = hvac.Client(
    url='http://127.0.0.1:8200',
    ...
```

We will probably want to add that comment to the rest of the language examples as well, not just Python.